<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="L_Nombre">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="L_Apellido">
     <property name="text">
      <string>Apellido</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="L_Legajo">
     <property name="text">
      <string>Legajo</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="LE_Captcha"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="L_Captcha">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="L_CaptchaTitle">
     <property name="text">
      <string>Captcha</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="LE_Apellido"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="LE_Nombre"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="LE_Legajo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
